docs/styleguide.config.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  components: path.resolve(__dirname, '../react/{Button,Toggle}/*.{js,jsx}'),
+  webpackConfig: require('./webpack.config.js'),
+  serverPort: 6161,
+  styleguideDir: path.resolve(__dirname, '../build/react')
```

Comment:
I would add `showUsage: true` to show props and methods by default

![image](https://user-images.githubusercontent.com/465582/30370914-4830a0b4-9879-11e7-967c-94a9c9d5f939.png)

Reply:
as you wish, no pb